competency-insights-ui/src/components/Layout/Header.js

```diff
@@ -0,0 +1,72 @@
+import React, { useState, useEffect } from 'react';
+import { Link } from 'react-router-dom';
+
+export const Header = () => {
+
+    const [darkMode, setDarkMode] = useState(JSON.parse(localStorage.getItem("darkMode")) || false);
+    const [shouldDropDownOpen, setShouldDropDownOpen] = useState(false);
+    useEffect(() => {
+        localStorage.setItem("darkMode", JSON.stringify(darkMode));
+
+        if (darkMode) {
+            document.documentElement.classList.add("dark");
+        } else {
+            document.documentElement.classList.remove("dark");
+        }
+    }, [darkMode]);
+
+    return (
+        <header >
+            <nav className="bg-white dark:bg-gray-900">
+                <div className="border-b border-slate-200 dark:border-b-0 flex flex-wrap justify-between items-center mx-auto max-w-screen-xl px-4 md:px-6 py-3">
+
+                    <Link to="/" className="flex items-center">
+                        <img src="/nashtech_logo.png" className="h-8" alt="Logo" />
+                        <span className="text-2xl font-semibold whitespace-nowrap dark:text-white">Java Competency</span>
+                    </Link>
+
+                    {/* <form className="flex items-center max-w-sm mx-auto">
+
+                        <div className="relative w-full">
+
+                            <input type="text" id="simple-search" className="bg-gray-50 border border-gray-300 text-gray-900 text-sm rounded-lg focus:ring-blue-500 focus:border-blue-500 block w-full ps-10 p-2.5  dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-blue-500 dark:focus:border-blue-500" placeholder="Search employee name..." required />
```

Comment:
Does this commented code has some future functionality ?

Reply:
yes it is for search feature it can be implemented in dashboard.